__tests__/app.test.js

```diff
@@ -82,4 +83,42 @@ describe('Testing app', () => {
                 })
         });
     });
+    describe('GET /api/articles/:article_id', () => {
+        test('status: 200, responds with a single article', () => {
+          return request(app)
+            .get(`/api/articles/1`)
+            .expect(200)
+            .then(({ body: { article } }) => {
+                article.forEach((oneArticle) => {
+                  expect(oneArticle).toEqual(

```

Comment:
shouldn't be coming back in an array! It's just one article, I'd like an object please

Reply:
Removed the forEach and adjusted the code to bring back a single object. Test working.